src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -40,39 +37,49 @@ interface NewsletterSignupProps {
   id?: string;
   /** Toggles the invalid state for the email field. */
   isInvalidEmail?: boolean;
-  /** Value to determine the section color highlight. The default is set to "blogs" as it uses the
-   * "ui.border.deafult" color.
+  /** Value to determine the section color highlight.
    */
-  newsletterSignupType?: SectionTypes;
+  highlightColor?: HighlightColorTypes;
   /** A handler function that will be called when the form is submitted. */
   onSubmit: (event: React.FormEvent<any>) => void;
   /** A handler function that will be called when the text input changes. */
   onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Gives option to remove the default Privacy Link if a custom one is provided
+   * NOTE: A Privaly Policy link should always be included.
+   */
+  showDefaultPrivacyLink?: boolean;
```

Comment:
I think you can rename this `showPrivacyLink`

Reply:
Done.